Resources/Prototypes/Entities/Structures/Wallmounts/shelfs.yml

```diff
@@ -35,7 +33,6 @@
       - !type:DoActsBehavior
         acts: ["Destruction"]
   - type: WallMount
-    arc: 175
```

Comment:
isn't this going to make them accessible from either side of the wall?

Reply:
Tested upstream. Apparently it defaults to 180

Reply:
yes 180 is fine i think we had 175 for a while then it was upstreamed and changed to 180

Reply:
Really it should be 90, or you'll get occasional issues where you can access them through windoors/thindows but 🤷🏼‍♂️

Reply:
pretty sure changing it to 90 would break maps, and be really annoying

Reply:
It wouldn't because no one has mapped them yet.

But also....I take it back...175 *is* what is should be [I've changed this before](https://github.com/space-wizards/space-station-14/pull/25451/files)